README.md

```diff
@@ -299,7 +299,19 @@ Additionally, users can utilize the MiNiFi Toolkit Converter (version 0.0.1 - sc
 
     if you do not want to enable client certificate base authorization
     nifi.security.need.ClientAuth=false
-
+    
+### Configuring Volatile and NO-OP Repositories
+
+     in minifi.properties 
+     
+     # For Volatile Repositories:
+     nifi.flow.repository.class.name=VolatileRepository
+     nifi.provenance.repository.class.name=VolatileRepository
+     
+     # For NO-OP Repositories:
+	 nifi.flow.repository.class.name=NoOpRepository
```

Comment:
My thought, initially was that we could have processors that could be self contained and work with or without connections. In those cases the configuration wouldn't need to be updated but we may be able to disable both repositories at runtime thus altering certain devices to have a different capability. I support what you are suggesting though but wonder if the Java model fits all embedded devices and use cases.